webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/repository/jdbc/ExpectedReportsJdbcRepository.scala

```diff
@@ -211,47 +231,64 @@ class UpdateExpectedReportsJdbcRepository(
   }
 
 
-  def saveNodeExpectedReports(configs: List[NodeExpectedReports]): Box[List[NodeExpectedReports]] = {
-
+  def saveNodeExpectedReports(configs: List[NodeExpectedReports]): Box[Seq[NodeExpectedReports]] = {
```

Comment:
this fails with 
[2019-10-31 15:03:21] ERROR policy.generation - Error when updating policy, reason was: Error when saving expected reports <- Batch entry 0 
                         insert into nodes_info (config_ids, node_id) values ('[{"2019-10-31T14:52:53.287Z":"20191031-145253-21773e69"}]', 'e970cc21-e176-4315-9e05-671468efab5b')
                        was aborted: ERROR: duplicate key value violates unique constraint "nodes_info_pkey"